<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MagCalForm</class>
 <widget class="QWidget" name="MagCalForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>969</width>
    <height>668</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>外磁计算</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>951</width>
     <height>661</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>13</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>外磁选择：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbxMag">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>9</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>303</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>最小值：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtMin">
    <property name="geometry">
     <rect>
      <x>71</x>
      <y>301</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pbCut">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>327</x>
      <y>298</y>
      <width>81</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>截取</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbRecToFile">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>620</y>
      <width>141</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>记录数据到本地</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pbSaveMag">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>754</x>
      <y>614</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>计算并写入飞控</string>
    </property>
   </widget>
   <widget class="MyCurve" name="curveSrc">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>41</y>
      <width>931</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtMax">
    <property name="geometry">
     <rect>
      <x>235</x>
      <y>301</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>174</x>
      <y>303</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>最大值：</string>
    </property>
   </widget>
   <widget class="MyCurve" name="curveCut">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>335</y>
      <width>931</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labShow">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>306</y>
      <width>371</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyCurve</class>
   <extends>QListView</extends>
   <header>mycurve.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
